src/components/TreeGrid.ts

```diff
@@ -147,13 +153,27 @@ const TreeGridInner = <T extends RowContents>(props: TreeGridPropsInner<T>) => {
 
   const visibleRows = getVisibleRows(data);
 
+  const rowsShowing = height ? height / rowHeight : 0;
+  // We want to place the selected concept in the middle of the visible rows, so we scroll to the element plus half the rows showing
```

Comment:
Same comment as before; also the shown row is not necessarily selected and TreeGrid itself doesn't have a concept of selected rows.
```suggestion
  // We want to place the shown row in the middle of the visible rows, so we scroll to the element plus half the rows showing
```